python_file.py

```
@@ -0,0 +1,213 @@
+            user = self._get_user_details(request.user.id)
+            data = request.data
+
+            category_name = data.get('category_name', '').strip()
+
+            self._send_notification(user, category_name)
+
+            return Response(
+                {'message': 'Lead created successfully'},
+                status=status.HTTP_201_CREATED,
+            )
+        except Exception as e:  # pylint: disable=broad-exception-caught
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)